README.rst

```diff
@@ -128,31 +128,19 @@ Browser Extensions
 ^^^^^^^^^^^^^^^^^^
 Run the following command at the prompt to build the Chrome extension::
 
-    $ ./bin/hypothesis extension development.ini chrome http://localhost:5000
+    $ hypothesis-buildext development.ini chrome
 
 Or, to load the assets from within the extension::
 
-    $ ./bin/hypothesis extension development.ini chrome http://localhost:5000 \
-    chrome-extension://extensionid/public
+    $ hypothesis-buildext development.ini chrome --base http://localhost:5000 --assets chrome-extension://extensionid/public
 
 To build an extension with a feature flag enabled use the environment variable::
 
-    FEATURE_NOTIFICATION=true \
-    hypothesis extension production.ini chrome \
-    https://hypothes.is chrome-extension://extensionid/public
+    $ FEATURE_NOTIFICATION=true hypothesis-buildext production.ini chrome --base https://hypothes.is --assets chrome-extension://extensionid/public
```

Comment:
Never mind. I forgot there are the notification settings in the user preferences.